index.js

```diff
@@ -32,24 +32,27 @@ module.exports = function(tilelive, options) {
     }
 
     // TODO pass scale
-    return async.map(sourceUris, async.apply(tilelive.load), function(err, sources) {
-      if (err) {
-        return callback(err);
-      }
-
+    return async.reduce(sourceUris, [], function(sources, uri, next) {
+      tilelive.load(uri, function(err, source) {
+        if (!err) sources.push({ uri: uri, source: source });
+        return next(null, sources);
+      })
+    }, function(err, sources) {
+      if (souces.length === 0) return callback(new Error("Not found any valid sources");
```

Comment:
Typo: `souces` → `sources`

Also, please add braces for style and change the error message to "Did not find any valid sources".